static/javascript/launch.js

```diff
@@ -262,13 +271,20 @@ function includeSurveyMetadataFields(schema_name, survey_type) {
     .querySelector("#survey-type-metadata-accordion")
     .classList.remove("ons-u-vh");
   document.querySelector(".survey_heading").innerHTML =
-    `${survey_type} Survey Metadata`;
+    `${escapeHtml(survey_type)} Survey Metadata`;
 
-  document.querySelector("#survey_metadata_fields").innerHTML =
-    `<div class="ons-field ons-field--inline">
-      <label class="ons-label" for="form_type">form_type</label>
-      <input id="form_type" name="form_type" type="text" value="${formTypeValue}" class="ons-input ons-input--text ons-input-type__input">
-    </div>`;
+  const surveyMetadataFields = document.querySelector(
```

Comment:
According to the alert this is one of many instances where we will need a fix?

Reply:
yeah there are two alerts in this file. This change is suggested from here:
https://github.com/ONSdigital/eq-questionnaire-launcher/security/code-scanning/8

Reply:
There are actually 18 changes suggested when you press "show paths"?

Reply:
I have had a look. I think it is just showing the path leading to the error. As you can see at the end of the path is what the codeQL suggests fixing.

Reply:
Ok, so fixing the issue from the last step in both alerts will clear them?

Reply:
Yeah, thats what it looks like. as only the parts i have changed have dom text issues